### PR TITLE
feat: added custom `not-found` or 404 pages

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:**

- This PR adds custom 404 pages for the `web` & `marketing` app

**Web App**

Screenshot 2023-09-03 at 11 53 22 PM

**Marketing App**

Screenshot 2023-09-03 at 11 53 30 PM

### CODE REVIEW

**Comment by adithyaakrishna** [comment]
The current 404 page looks like this

Screenshot 2023-09-04 at 12 01 14 AM

**Comment by dephraiim** [comment]
Can you add the logo?

**Comment by adithyaakrishna** [comment]
Yes, I was thinking of animating the logo. Kinda makes it look cool. What say? 👀

File: apps/marketing/src/app/not-found.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+export default function NotFound() {
+  const router = useRouter();
+  return (
+    <motion.div className={cn('relative')}>
+      <div className="absolute -inset-24 -z-10">
+        <motion.div
+          className="flex h-full w-full origin-top-right items-center justify-center"
+          initial="initial"
+          animate="animate"
```
Comment (Mythie) [inline apps/marketing/src/app/not-found.tsx:22]: We haven't passed in any variants so these two wouldn't be doing anything right now 🤔

**Review by Mythie — APPROVED** [review]

File: apps/marketing/src/app/not-found.tsx (hunk 1 above)
Comment (Mythie) [reply apps/marketing/src/app/not-found.tsx]: Updated on my end, cheers!

**Comment by Mythie** [comment]
Thanks Adii! 🥳

**Pull request merged**